src/Utils/axiosRetry.ts

```diff
@@ -8,6 +11,13 @@ import IAxiosRetryOptions from "../Interfaces/IAxiosRetryOptions";
  * @param axiosRetryOptions Options for axios retry.
  */
 const axiosRetry = (axios: AxiosInstance, axiosRetryOptions: IAxiosRetryOptions) => { // eslint-disable-line @typescript-eslint/explicit-module-boundary-types
+  const retryInterval = 1000; // 1 second interval between retries
+
+  // Default values
+  if (axiosRetryOptions.retryOn429 === undefined || axiosRetryOptions.retryOn429 === null) {
+    axiosRetryOptions.retryOn429 = true;
```

Comment:
I think default should be false, only true if it is explicitly defined

Reply:
`axiosRetry` is being used on every HTTP calls. No retry on 429 is only applicable on `getchattoken`. So, `retryOn429` should be `true` by default to keep backward compatibility.